pkgs/development/python-modules/hyperpyyaml/default.nix

```diff
@@ -0,0 +1,39 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytestCheckHook
+, pyyaml
+, ruamel-yaml
+}:
+
+buildPythonPackage rec {
+  pname = "hyperpyyaml";
+  version = "1.2.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "speechbrain";
+    repo = "hyperpyyaml";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
From my experience, @SuperSandro2000 recommends not using `pname` here but hardcoding the repo name instead.